packages/dev-utils/src/web3_factory.ts

```diff
@@ -41,6 +41,7 @@ export const web3Factory = {
             }
             provider.addProvider(
                 new GanacheSubprovider({
+                    gasLimit: constants.GAS_ESTIMATE,
```

Comment:
Let's rename `GAS_ESTIMATE` to `GAS_LIMIT`. `estimate` can also refer to gasAmount.